core/shared/src/main/scala/zio/Semaphore.scala

```diff
@@ -96,17 +96,31 @@ final class Semaphore private (private val state: Ref[State]) extends Serializab
   }
 
   /**
-   * Acquires a permit, executes the action and releases the permits right after.
+   * Acquires a permit, executes the action and releases the permit right after.
    */
   final def withPermit[R, E, A](task: ZIO[R, E, A]): ZIO[R, E, A] =
-    withPermits(1L)(task)
+    withPermits(1)(task)
+
+  /**
+   * Acquires a permit in a [[zio.ZManaged]], executes the action and releases the
+   * permit in the finalizer.
+   */
+  final def withPermitManaged[R, E]: ZManaged[R, E, Unit] =
+    withPermitsManaged(1)
 
   /**
    * Acquires `n` permits, executes the action and releases the permits right after.
    */
   final def withPermits[R, E, A](n: Long)(task: ZIO[R, E, A]): ZIO[R, E, A] =
     prepare(n).bracket(_.release)(_.awaitAcquire *> task)
 
+  /**
+   * Acquires `n` permits in a [[zio.ZManaged]], executes the action and releases the
```

Comment:
Sorry could you just update the scaladoc?

Reply:
You're right, sorry.

Reply:
Done.